devutils/update_lists.py

```diff
@@ -43,6 +43,12 @@
     # Exclusions for DOM distiller (contains model data only)
     'components/dom_distiller/core/data/distillable_page_model_new.bin',
     'components/dom_distiller/core/data/long_page_model.bin',
+    # Exclusions for GeoLanguage data
+    # Details: https://docs.google.com/document/d/18WqVHz5F9vaUiE32E8Ge6QHmku2QSJKvlqB9JjnIM-g/edit
+    # Introduced with: https://chromium.googlesource.com/chromium/src/+/6647da61ef03f4faec45a31e8ef6d6e4968d87bd
```

Comment:
This is breaking CI checks. Could you break this up into two lines?

Reply:
Done.